tests/integration/test_badge_escaping.bats

```diff
@@ -169,12 +173,12 @@ teardown() {
     [[ "$bad_url" =~ "label=Build_&_Test" ]]  # & creates new query param - BROKEN
 }
 
-@test "CI badge: label with space should use jq @uri for query param" {
+@test "CI badge: label with space should use url_encode() for query param" {
     # Arrange
     local ci_label="Build Status"
 
     # Act
-    local jq_escaped=$(jq -rn --arg s "${ci_label}" '$s | @uri')
+    local jq_escaped=$(url_encode "${ci_label}")
```

Comment:
Maybe you could find a better name for this variable now

`jq_escaped` when `jq` is no longer used sounds awkward

This maybe

```suggestion
    local ci_label_escaped=$(url_encode "${ci_label}")
```

Reply:
Good catch !